lms/resources/_js_config.py

```diff
@@ -203,9 +204,6 @@ def _add_canvas_speedgrader_settings(self, **kwargs):
 
     def _auth_token(self):
         """Return the authToken setting."""
-        if not self._lti_user:
-            return None
-
```

Comment:
I don't think `LTILaunchResource` is ever used when `request.lti_user` is `None` so I've just simplified this